hugegraph-test/src/main/java/com/baidu/hugegraph/api/ProjectApiTest.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2017 HugeGraph Authors
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.baidu.hugegraph.api;
+
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.common.base.Strings;
+
+public class ProjectApiTest extends BaseApiTest {
+
+    private final static String path = "graphs/hugegraph/auth/projects";
+
+    @Before
+    public void setUp() {
+        BaseApiTest.truncate();
+    }
+
+    @Test
+    public void testCreate() {
+        createProject("test_project", "this is a good project");
+    }
+
+    public String createProject(String name, String desc) {
```

Comment:
set private static and move the method to the bottom of the class

Reply:
this method need use client that is a field of object, so may i replace public to private